test/test_pipeline/components/test_tabular_column_transformer.py

```diff
@@ -12,6 +12,11 @@
     TabularColumnTransformer
 )
 
+@pytest.mark.parametrize("fit_dictionary", ['fit_dictionary_numerical_only',
```

Comment:
I dont think this change is correct.

Reply:
I did not change, but it was merged from the latest change (so I will get it back to the original)